bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RoundRobinDistributionSchedule.java

```diff
@@ -60,6 +60,11 @@ public WriteSet getWriteSetForLongPoll(long entryId) {
         return WriteSetImpl.create(ensembleSize, ensembleSize /* writeQuorumSize */, entryId);
     }
 
+    @Override
+    public WriteSet getWriteSetForForceLedger() {
+        return WriteSetImpl.create(ensembleSize, ensembleSize /* writeQuorumSize */, 0);
```

Comment:
Have a comment similar to longPoll method above.

Reply:
done